TLM/CSUtil.Commons/Log.cs

```diff
@@ -163,16 +163,9 @@ private enum LogLevel {
             LogToFile(string.Format(format, args), LogLevel.Error);
         }
 
-        /// <summary>
-        /// Log an error only if cond is true
-        /// NOTE: If a lambda contains values from `out` and `ref` scope args,
-        /// then you can not use a lambda, instead use `if (cond) { Log.Error() }`
-        /// </summary>
-        /// <param name="cond">The condition</param>
-        /// <param name="s">The function which returns text to log</param>
-        public static void ErrorIf(bool cond, Func<string> s) {
+        public static void ErrorIf(bool cond, string s) {
```

Comment:
i think there the reason this accepted function instead of string was because the author was trying to avoid performing string formatting in debug builds.

are you sure this does not reduce performance or allocate garbage memory in if there is no error?

in any case the lambda to generate formatted string created garbage in the previous code anyway.

Reply:
Both versions with lambda and with `$""` create garbage and lambda will do less work in case when condition is false.
I can probably revert it to Lambda variant.
This function is not used in critical code paths, only used in 2 locations (Timed TL: ChangeLightMode, and in U UI library if an exception happened during a resize event.

Reply:
Reverted to lambda